tide/master/localstreamer/CommandLineOptions.cpp

```diff
@@ -40,175 +40,194 @@
 #include "CommandLineOptions.h"
 
 #include <iostream>
-#include <boost/program_options.hpp>
 #include <QStringList>
+#include <boost/program_options.hpp>
+
+namespace po = boost::program_options;
 
 CommandLineOptions::CommandLineOptions()
-    : getHelp_(false)
-    , streamerType_(PS_UNKNOWN)
-    , width_(0)
-    , height_(0)
-    , desc_("Allowed options")
+    : _getHelp( false )
+    , _streamerType( PS_UNKNOWN )
+    , _width( 0 )
+    , _height( 0 )
+    , _desc( "Allowed options" )
 {
-    initDesc();
+    _initDesc();
 }
 
-CommandLineOptions::CommandLineOptions(int &argc, char **argv)
-    : getHelp_(false)
-    , streamerType_(PS_UNKNOWN)
-    , width_(0)
-    , height_(0)
-    , desc_("Allowed options")
+CommandLineOptions::CommandLineOptions( int& argc, char** argv )
+    : _getHelp( false )
+    , _streamerType( PS_UNKNOWN )
+    , _width( 0 )
+    , _height( 0 )
+    , _desc( "Allowed options" )
 {
-    initDesc();
-    parseCommandLineArguments(argc, argv);
+    _initDesc();
+    _parseCommandLineArguments( argc, argv );
 }
 
-void CommandLineOptions::initDesc()
+void CommandLineOptions::_initDesc()
 {
-    desc_.add_options()
+    _desc.add_options()
         ("help", "produce help message")
-        ("name", boost::program_options::value<std::string>()->default_value(""),
-                 "unique identifier for this stream")
-        ("type", boost::program_options::value<std::string>()->default_value(""),
-                 "streamer type [webkit | dock]")
-        ("width", boost::program_options::value<unsigned int>()->default_value(0),
-                 "width of the stream in pixel")
-        ("height", boost::program_options::value<unsigned int>()->default_value(0),
-                 "height of the stream in pixel")
-        ("url", boost::program_options::value<std::string>()->default_value(""), "webkit only: url")
-        ("rootdir", boost::program_options::value<std::string>()->default_value(""), "dock only: root directory")
+        ("streamname", po::value<std::string>()->default_value( "" ),
+         "unique identifier for this stream")
+        ("type", po::value<std::string>()->default_value( "" ),
+         "streamer type [webkit | dock]")
+        ("width", po::value<unsigned int>()->default_value( 0 ),
+         "width of the stream in pixel")
+        ("height", po::value<unsigned int>()->default_value( 0 ),
+         "height of the stream in pixel")
+        ("url", po::value<std::string>()->default_value( "" ),
+         "webkit only: url")
+        ("rootdir", po::value<std::string>()->default_value( "" ),
+         "Dock only: root files directory")
```

Comment:
Not used by the launcher so will be removed. the launcher reads the configuration.xml instead of passing a ton of command line paramterers